kmk/internal_state.py

```diff
@@ -47,10 +47,26 @@ def _to_dict(self):
         return ret
 
     def _find_key_in_map(self, row, col):
+        ic = intify_coordinate(row, col)
+
+        try:
+            idx = self.config.coord_mapping.index(ic)
```

Comment:
This is technically super inefficient (introduces an `O(n)` lookup where we once had an `O(1)`, but `coord_mapping` will need to change to some form of map/dict to bring this back to an `O(1)`. This can likely be done as a compile-down at firmware boot time, much like what we do with leader key sequences.